lib/index.js

```diff
@@ -46,19 +49,26 @@ class RtspClient extends EventEmitter {
 
         this.packetLength = -1;
 
-        const packet = parseRTPPacket(Buffer.concat(this.packets));
-        if (packet.length + 16 !== this.originalPacketLength) {
-          throw new Error("Bug in RTSP data framing, please file an issue with the author w/ stacktrace.");
+        let packet;
+        if (this.packet_channel_id === 0) {
+          packet = parseRTPPacket(Buffer.concat(this.packets));
+          if (packet.length + 16 !== this.originalPacketLength) {
+            throw new Error("Bug in RTSP data framing, please file an issue with the author w/ stacktrace.");
+          }
+        }
+        if (this.packet_channel_id === 1) {
+          packet = parseRTCPPacket(Buffer.concat(this.packets));
         }
 
         this.packets = [];
 
-        this.emit("data", packet.payload, packet);
+        this.emit("data", this.packet_channel_id, packet.payload, packet);
```

Comment:
Is this really the cleanest thing, API wise, that we could do in this situation? Since from what I can see from RTSP specification this will only ever be zero or one, could a separate event for RTCP, like maybe "controlData", be fired instead? I think it would remove some possible confusion.